content/documentation/figure-images.md

```diff
@@ -156,6 +156,25 @@ Quire supports video embeds from either YouTube (`media_type: youtube`) or Vimeo
 
 {{< /q-class >}}
 
+## Add Tables
+
+Tables can be embedded in your publication as HTML files. If your table exists in a Word document, resave the Word document as an .html file. In the `static/image/` directory of your project create a new folder called `tables`. Place the .html files in that folder.
+
+Similarly, to embedding videos, you will need to add a `media_type` to the entry in the `figures.yml`. In this case, the type would be `table`. Make sure the `src` has the .html suffix rather than .jpg.
+
+```YAML
+- id: "table-1-1"
+  src: "tables/table-1-1.html"
+  media_type: table
+  label: "Table 1.1"
+```
+
+Then use the `q-figure` shortcode as normal to insert the table in to the page:
+
+```go
+{{</* q-figure id="table-1-1" */>}}
```

Comment:
Do we also want to mention that users could add simple tables using Markdown's built-in table structuring, but that the tables won't be zoomable like other figures?

Reply:
Ah yes! I think we should definitely.